src/QirRuntime/README.md

```diff
@@ -98,12 +98,42 @@ For tests that depend on the native simulator and qdk shared libraries, you migh
 
 This project contains an implementation of the QIR runtime per the
  [QIR specifications](https://github.com/microsoft/qsharp-language/tree/main/Specifications/QIR) and the translation
- layer between the QIR and the IR, generated by Clang from the native code. We call the translation layer "QIR Bridge".
- This project also provides an optional implementation of intrinsics from `quantum__qis*` namespace that are used in QIR,
- generated from Q#.
+ layer between the QIR and the IR, generated by Clang from the native code. Translation layer is called the "QIR Bridge".
 
 ![QIR Bridge architecture diagram](qir.png?raw=true "QIR Bridge architecture diagram")
 
+This project also provides an implementation of the quantum instruction set, used by Q# for simulation against the full
```

Comment:
From time to time I forget what QIS stands for. And I don't remember any doc explicitly telling what QIS is.
This README.md seems to me a good place to provide that info, and if the QIS stands for "quantum instruction set", then we could mention like this
> This project also provides an implementation of the quantum instruction set **(QIS)**,

Reply:
This naming convention depends on the compiler. For example, the math functions will likely use the same naming scheme and they aren't exactly part of the quantum instruction set... The subset of `__quantum__qis*` functions listed here is exactly the QIS supported by our simulators, though.